compose/core-compose/src/main/java/com/squareup/workflow/ui/compose/internal/ViewFactories.kt

```diff
@@ -92,7 +91,7 @@ import kotlin.properties.Delegates.observable
   // Plumb the current composition through the ViewEnvironment so any nested composable factories
   // get access to any ambients currently in effect.
   val parentComposition = remember { ParentComposition() }
-  parentComposition.reference = compositionReference()
+  parentComposition.reference = rememberCompositionContext()
```

Comment:
This line, and the `reference` property, and all the manual wiring up of parent compositions, can go away because of #300. You already deleted the consuming side above, and this can be deleted as well.